### ISSUE TITLE
Private campain templates

### ISSUE THREAD

**Issue by hmmmfctunlpt** [phpList/phplist3#1013]
Hello,

Thank you for the excelent work!

I'm using v3.6.14 and i believe there is either a bug or an improper way of handling campaign templates.
The way access to campaign templates work make it possible for any Admin user to edit any campaign template.
Rightfully, campaigns themselves, while private, are only available to the owner. I.e., the Admin that created the campaign. The problem arises when someone else, deliberatly or by accident, edits the template which is to be used by the other persons campaign, which could already be set to run.
I believe it would be beneficial to have a public/private context on campaign templates and restrict access (write or read/write) solely to the creator of the template.

Kind regards,

Hugo.

**Comment by michield** [comment]
Yes, I can see what you mean. Templates were never intended to be edited often. The idea is to have some standard branding for the organisation in it and some nice layout and then all the content is in the campaign. I've noticed people using the template functionality to actually write the entire campaign and put "" in the campaign content to get around the requirement to have some content. 

Ownership of templates would be a new feature, and can possibly be related to #929 where the request is to turn admins into admin-groups.
